src/containers/BooksList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+function BooksList({ books }) {
+  return (
+    <div>
+      <table border="1">
+        <thead>
+          <tr>
+            <th>Book ID</th>
+            <th>Book Title</th>
+            <th>Book Category</th>
+          </tr>
+        </thead>
+        <tbody>
+          {
+            books.map((book) => (
+              <Book
+                key={book.bookId}
+                bookId={book.bookId}
+                title={book.title}
+                category={book.category}
```

Comment:
- [ ] [OPTIONAL] I suggest that you pass a `book` object as a prop to each `Book` component like this:

`<Book key={book.bookId} book={book} />`

Reply:
I am still trying to figure out how to represent this in an html table